packages/calcite-components/src/components/input-text/input-text.scss

```diff
@@ -1,4 +1,43 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-input-text-corner-radius: defines the border radius of the component.
+ * @prop --calcite-input-text-text-color: defines the text color of the component.
+ * @prop --calcite-input-text-border-color: defines the border color of the component.
+ * @prop --calcite-input-text-background-color: defines the background color of the component.
+ * @prop --calcite-input-text-icon-color: defines the color of an icon element in the component.
+ * @prop --calcite-input-text-icon-color-hover: defines the color of an icon element when it's parent is hovered in the component.
+ * @prop --calcite-input-text-prefixsuffix-text-color: defines the text color of a prefix/suffix element in the component.
```

Comment:
Similar feedback to `input` - I think these would be better separate (prefix-text-color / background-color and suffix-text-color / background-color) as it makes the naming straightforward and IMO is a valid theming opportunity.